scripts/github-pr-title-lint.js

```diff
@@ -107,8 +107,8 @@ function getPullRequestTitle() {
         if (response.statusCode < 200 || response.statusCode > 299) {
           reject(
             new Error(
-              `getPullRequestTitle got an unexpected statusCode: ${response.statusCode}`
-            )
+              `getPullRequestTitle got an unexpected statusCode: ${response.statusCode}`,
+            ),
```

Comment:
@willdurand I find these two added commas in the Error constructor parameters a little odd, mainly because these are considered valid starting from ES2017, but in the end that shouldn't be a problem.